assets/src/editor/events/dates-and-times/editor-date/edit-form/use-date-entity-input-config.js

```diff
@@ -87,6 +102,24 @@ const useDateEntityInputConfig = ( dateEntity ) => useMemo( () => {
 					}
 				}
 			},
+			validate: ( value ) => {
+				if ( value ) {
+					const endDate = new Date( value );
+					const startDate = dateEntity.start.toJSDate();
+					if ( endDate - startDate < 0 ) {
+						return __( 'End time must be after start time.', 'event_espresso' );
```

Comment:
let's use text from the labels for the validation notices, so maybe something like :

```js
return __( 'End Date & Time must be after Start Date & Time.', 'event_espresso' );
```

or

```js
return __( 'End Date must be after Start Date.', 'event_espresso' );
```

Reply:
What about this? 😉
```JS
return sprintf( __( '%1$s must be after %2$s.', 'event_espresso' ), __( 'End Date & Time', 'event_espresso' ), __( 'Start Date & Time', 'event_espresso' ) );
```

Reply:
hmmmmm...

I'm leery about imposing grammatical structure like that because you never know if some other language has really different rules that would make translating that difficult. As well, some people will often use translations to simply modify the UI text to their liking, although I doubt that would ever be the case with some admin UI text like this.

that said, I do like that this reuses the two existing translations which means we can reuse this with the ticket sale dates as well.

I'm going to get JOsh's opinion on this in chat...